README.md

```diff
@@ -235,7 +230,6 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [try.cr](https://github.com/maiha/try.cr) - Try monad
 
 ## Feature Flipping
- * [can_use](https://github.com/rodrigopinto/can_use) - It is a minimalist feature toggle/flag for crystal based on yaml
```

Comment:
the section can be removed

Reply:
can_use was built for personal use, not intention it becomes used by a large audience, but would be better to keep the "Feature Toggle" section at least to promote other solutions like https://github.com/metaware/flipper?

Reply:
Flipper's latest commit was done 2 years ago. I would still suggest removing the section completely.